index.js

```diff
@@ -374,6 +376,36 @@
     return Nothing();
   });
 
+  //. indexOf :: a -> [a] -> Maybe Number
+  //.
+  //. Returns Just the position of the first occurrence of an
+  //. element in a list, or Nothing if the element does not occur
+  //. in the list.
+  //.
+  //. ```javascript
+  //. > S.indexOf('c', ['a','b', 'c', 'd', 'e'])
+  //. Just(2)
+  //. ```
+  S.indexOf = R.curry(function(target, list) {
+    var idx = R.indexOf(target, list);
+    return indexOfToMaybe(idx);
+  });
```

Comment:
If we :curry: `filter` and add `var _ = R.__;` we can write:

```javascript
S.indexOf = R.curry(R.compose(filter(R.gte(_, 0)), Just, R.indexOf));
```

This was we could avoid having to come up with a good name for `indexOfToMaybe`. :)